resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/security/YARNHadoopDelegationTokenManager.scala

```diff
@@ -44,6 +44,8 @@ private[yarn] class YARNHadoopDelegationTokenManager(
 
   // public for testing
   val credentialProviders = getCredentialProviders
+  logDebug(s"Using the following YARN-specific credential providers: " +
+    s"${credentialProviders.keys.mkString(", ")}.")
```

Comment:
I think usually this will be empty, we should only output this log when there is credential provider loaded in.

Reply:
Sure.